src/databricks/labs/ucx/source_code/files.py

```diff
@@ -150,3 +150,29 @@ def resolve_import(self, path_lookup: PathLookup, name: str) -> MaybeDependency:
 
     def __repr__(self):
         return "LocalFileResolver()"
+
+
+class SitePackageResolver(BaseDependencyResolver):
+
+    def __init__(
+        self,
+        file_loader: FileLoader,
+        site_packages_path: Path,
+        next_resolver: BaseDependencyResolver | None = None,
+    ):
+        super().__init__(next_resolver)
+        self._file_loader = file_loader
+        self._site_packages_path = site_packages_path
+
+    def with_next_resolver(self, resolver: BaseDependencyResolver) -> BaseDependencyResolver:
+        return SitePackageResolver(self._file_loader, self._site_packages_path, resolver)
+
+    def resolve_import(self, path_lookup: PathLookup, name: str) -> MaybeDependency:
+        path = Path(self._site_packages_path, name)
+        if not path.is_dir():
+            return super().resolve_import(path_lookup, name)
+        path = Path(path, "__init__.py")
+        if not path.is_file():
+            return super().resolve_import(path_lookup, name)
+        dependency = Dependency(self._file_loader, path)
```

Comment:
We are not using like the PipResolver does

Reply:
I would expect a similar output

Reply:
Not sure I understand your comment.